packages/web/src/scss/components/UNSTABLE_Slider/README.md

```diff
@@ -1,4 +1,233 @@
+    aria-describedby="slider-warning-validation-text"
+    style="--slider-position: 30%"
+    type="range"
+    value="30"
+    oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 100)}%`);"
+  />
+  <div id="slider-warning-validation-text" class="UNSTABLE_Slider__validationText">Validation text</div>
+</div>
+
+<div class="UNSTABLE_Slider UNSTABLE_Slider--danger">
+  <label for="slider-danger" class="UNSTABLE_Slider__label">Slider</label>
+  <input
+    class="UNSTABLE_Slider__input"
+    id="slider-danger"
+    aria-describedby="slider-danger-validation-text"
+    style="--slider-position: 30%"
+    type="range"
+    value="30"
+    oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 100)}%`);"
+  />
+  <ul id="slider-danger-validation-text" class="UNSTABLE_Slider__validationText">
+    <li>First validation text</li>
+    <li>Second validation text</li>
+  </ul>
+</div>
+```
+
+### JavaScript-Controlled Validation Text
+
+When implementing client-side form validation, use JS interaction state classes
+(`has-success`, `has-warning`, `has-danger`) on the wrapping `<div>` element and
+render validation texts in a `<div>` with `data-spirit-element="validation_text"`
+attribute. This way your JS remains disconnected from CSS that may or may not be
+[prefixed].
+
+**Remember this approach is only valid for vanilla JS implementation. React
+components mix CSS with JS by design and handle prefixes their own way.**
+
+```html
+<div class="UNSTABLE_Slider UNSTABLE_Slider--danger">
+  <label for="slider-danger" class="UNSTABLE_Slider__label">Slider</label>
+  <input
+    class="UNSTABLE_Slider__input"
+    id="slider-danger"
+    aria-describedby="slider-danger-validation-text"
+    style="--slider-position: 30%"
+    type="range"
+    value="30"
+    oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 100)}%`);"
+  />
+  <div data-spirit-element="validation_text">Error message inserted by JS</div>
+</div>
+```
+
+## Disabled State
+
+On top of adding the `disabled` attribute to the input, disabled Slider can
+be marked by adding `UNSTABLE_Slider--disabled` modifier class, or with `is-disabled`
+JS interaction class when controlled by JavaScript:
+
+```html
+<div class="UNSTABLE_Slider UNSTABLE_Slider--disabled">
+  <label for="slider-disabled" class="UNSTABLE_Slider__label">Slider</label>
+  <input
+    class="UNSTABLE_Slider__input"
+    id="slider-disabled"
+    style="--slider-position: 30%"
+    type="range"
+    value="30"
+    oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 100)}%`);"
+    disabled
+  />
+</div>
+```
+
+[mdn-range]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range
+[html-spec-range]: https://html.spec.whatwg.org/multipage/input.html#range-state-(type=range)
+[prefixed]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web#prefixing-css-class-names
```

Comment:
shouldn't we link the README file directly? I know this works, but I am not sure if we should trust it :)